main/utils/devices.js

```diff
@@ -18,16 +18,28 @@ const getAudioDevices = async () => {
       const Sentry = require('./sentry');
       Sentry.captureException(new Error(`devices is not an array: ${JSON.stringify(devices)}`));
 
-      return (await audioDevices.getInputDevices()).map(device => ({id: device.uid, name: device.name}));
+      return (await audioDevices.getInputDevices()).map(device => ({id: device.uid, name: device.name})).sort(devicesSort);
```

Comment:
Shouldn't this be changed in `macos-audio-devices` instead?

Reply:
Hmm, but the package in general, doesn't really have a use-case for sorting them. I feel like that's a Kap-specific requirement